client/index.html

```diff
@@ -94,31 +94,30 @@ <h1 class="title">Connect</h1>
 									<h2>Network settings</h2>
 								</div>
 								<div class="col-sm-3">
-								<label>Name</label>
-									</div>
+									<label for="connect:name">Name</label>
+								</div>
 								<div class="col-sm-9">
-									<input class="input" name="name" value="<%= defaults.name %>">
+									<input class="input" id="connect:name" name="name" value="<%= defaults.name %>">
```

Comment:
For future references, I think we should avoid using `:` in `id`s. I believe they need escaping if we want to grab them with jQuery or CSS, and in CSS it would look confusing as `:` is used for pseudo-elements.

I get the idea of namespacing though, and I think using `-` or `_` for that works fine too.